content/en/apps/guides/debugging/obtaining-logs.md

```diff
@@ -1,27 +1,63 @@
+
+   ![Allow USB debugging](/apps/guides/debugging/images/allow_usb_debugging.png)
+
+#### Browser logs
+
+Now that you've enabled USB debugging on the phone you can access the dev console on the phone via USB from your desktop browser. This allows the same level of debugging and inspection you have on a desktop browser, but from your phone's browser in the CHT app. Follow the [official Android documentation](https://developer.chrome.com/docs/devtools/remote-debugging/webviews/#open_a_webview_in_devtools) to access the console.
+
+#### Android logs
 
-The easiest way is to use `cht-logs`, a tool that comes with [`cht-conf`](https://github.com/medic/cht-conf):
+The Android log is written to from the cht-android wrapper which can be useful to debug application crashes or failing integrations between apps.
 
+1. To install the `adb` command, follow the instructions under the [Development Environment > Debug tool adb]({{< ref "core/guides/android/development-setup#debug-tool-adb" >}}) section.
+2. Within a command line session, write the following command: `adb start-server`.
+3. To check if your phone is properly connected, write the command `adb devices`. This will list the devices connected.
+
+   ![ADB Devices](/apps/guides/debugging/images/adb_devices.png)
+
+4. Type the command `adb logcat > phone.log` to get the android logs in the file `phone.log`. You can stop the recording at any time pressing _Ctrl+C_. Now while taking android logs is the time to try to reproduce the error. Do this for as many times as you deem necessary to capture logs for that event.
+
+This will get all the logs from the device, not just logs for the app you want to debug. You can pass arguments to `adb` to only get the logs you want.
+
+```sh
+$ adb logcat MedicMobile:V AndroidRuntime:E chromium:V '*:S' > phone.log
 ```
-cht-logs anInstance gardener
+
+Alternatively you can use `grep` to filter the logs down to only those from the relevant app.
+
+```sh
+$ adb logcat | grep MedicMobile > phone.log
 ```
 
-See [cht-logs documentation](https://github.com/medic/cht-conf#user-content-fetching-logs) for more details.
+### On the server
+
+Some unexpected errors are caught and stored in `feedback` docs and stored on the phone and later synced to CouchDB on the server. To access these, look for docs in the `medic-user-{username}-meta` or `medic-users-meta` databases. This is particularly useful to debug issues where you do not have physical access to the device. More information is available under [Managing Databases]({{< relref "apps/guides/database" >}}).
+
+## Server logs
 
-This will download logs to your current directory. You will need to look through these logs to work out what is relevant.
\ No newline at end of file
+To access the server logs you will need access to the server.
```

Comment:
Wow, you're right, this is terribly redundant. It'll also change a little with CHT v4. The reason I dropped the documentation on the cht-logs command is it only worked on v2.

I'm tempted to drop this altogether until we have something useful to put here. What do you think? If I do that do I have to rename this whole file to be client centric (ie: drop the word server from everywhere)?

Reply:
Yup - I say drop it all together

Reply:
Done